builder/builder-next/adapters/containerimage/pull.go

```diff
@@ -945,7 +950,7 @@ func applySourcePolicies(ctx context.Context, str string, spls []*spb.Policy) (s
 		},
 	}
 
-	mut, err := sourcepolicy.NewEngine(spls).Evaluate(ctx, op)
```

Comment:
This can be simplified to

```go
	op := &pb.SourceOp{
		Identifier: srctypes.DockerImageScheme + "://" + ref.String(),
	}

	mut, err := sourcepolicy.NewEngine(spls).Evaluate(ctx, op)
```